features/pageobjects/dynamic.loading.page.js

```diff
@@ -0,0 +1,38 @@
+const { $ } = require('@wdio/globals')
+const Page = require('./page');
+
+
+class DynamicLoadingPage extends Page {
+    
+    get Example2Link () {
+        
+        return $('a[href="/dynamic_loading/2"]');

```

Comment:
locator is more stable, yeah, but still not parametrized
do something like exampleLink(number) ```
{
return $(`a[href="/dynamic_loading/${number}\`) 
}
```
to make it extendable